modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/Episode.kt

```diff
@@ -53,7 +53,7 @@ data class Episode(
     // Removed but leaving the column until we definitely aren't using it again
     @ColumnInfo(name = "cleanTitle") var cleanTitle: String? = title,
     @ColumnInfo(name = "last_playback_interaction_date") var lastPlaybackInteraction: Long? = null,
-    @ColumnInfo(name = "last_playback_interaction_sync_status") var lastPlaybackInteractionSyncStatus: Long = 1,
+    @ColumnInfo(name = "last_playback_interaction_sync_status") var lastPlaybackInteractionSyncStatus: Long = LAST_PLAYBACK_INTERACTION_SYNCED,
```

Comment:
I moved the history sync logic and switched this value to a constant at the same time to clean up.